backend/lib/teleservices/demarches-simplifiees.ts

```diff
@@ -47,9 +48,137 @@ const sources = {
   },
   situation: () => `J'ai ${"xx"} ans et je suis ${"yy"}`,
+  situation_familliale: (simulation) => {
+    const en_couple =
+      getAnswer(simulation.answers.current, "famille", "en_couple") === true
+
+    if (!en_couple) {
+      return "Célibataire"
+    }
+    const statut_marital = getAnswer(
+      simulation.answers.current,
+      "individu",
+      "statut_marital",
+      "conjoint"
+    )
+
+    // - Answer "en_couple" value "Union libre" is "celibataire" and needs to be restored as "Union libre" for the prefill
+    // - Missing not available answers in the simulator : Veuf(ve), Séparé(e), Divorcé(e)
+    const situations_couple = {
+      marie: "Marié(e)",
+      pacse: "Pacsé(e)",
+      celibataire: "Union libre",
+    }
+
+    return situations_couple[statut_marital]
+  },
   telephone: () => "0612345678",
 }
 
+const fsl_var_sources = {
+  activite: (simulation) => {
+    const activite = getAnswer(
+      simulation.answers.current,
+      "individu",
+      "activite",
+      "demandeur"
+    )
+    return getActiviteLabel(activite)
+  },
+  loyer_avec_charges: (simulation) => {
+    const { loyer, charges_locatives } = getAnswer(
+      simulation.answers.current,
+      "menage",
+      "loyer"
+    )
+    return loyer + charges_locatives
+  },
+  commune_nom: (simulation) => {
+    const { _nomCommune } = getAnswer(
+      simulation.answers.current,
+      "menage",
+      "depcom"
+    )
+    return _nomCommune
+  },
+  commune_code_postal: (simulation) => {
+    const { _codePostal } = getAnswer(
+      simulation.answers.current,
+      "menage",
+      "depcom"
+    )
+    return _codePostal
+  },
+  buildConjoint(simulation) {
+    if (
+      getAnswer(simulation.answers.current, "famille", "en_couple") === false
+    ) {
+      return null
+    }
+
+    const conjoint = {}
+    const date_naissance = getAnswer(
+      simulation.answers.current,
+      "individu",
+      "date_naissance",
+      "conjoint"
+    )
+    const activite = getAnswer(
+      simulation.answers.current,
+      "individu",
+      "activite",
+      "conjoint"
+    )
+
+    conjoint["champ_Q2hhbXAtMjU1NDk1MQ"] = date_naissance.slice(0, 10)
+    conjoint["champ_Q2hhbXAtMjU1NDk1NA"] = "Conjoint(e)"
+    conjoint["champ_Q2hhbXAtMjU1NDk1NQ"] = getActiviteLabel(activite)
+    return conjoint
+  },
+  buildChild(simulation, id) {
+    const child = {}
+    const date_naissance = getAnswer(
+      simulation.answers.current,
+      "individu",
+      "date_naissance",
+      `enfant_${id}`
+    )
+    const prenom = getAnswer(
+      simulation.answers.current,
+      "individu",
+      "_firstName",
+      `enfant_${id}`
+    )
+    const scolarite = getAnswer(
+      simulation.answers.current,
+      "individu",
+      "scolarite",
+      `enfant_${id}`
+    )
+    const scolariteLabel = Scolarite.types.find(
+      (t) => t.value === scolarite
+    )?.label
+    child["champ_Q2hhbXAtMjU1NDk1MQ"] = date_naissance.slice(0, 10)
```

Comment:
Pas bloquant, mais sache qu'en JS il vaut mieux construire l'objet en un coup, plutôt que de ajouter des champs à chaque fois (quitte à les créer sans valeur d'ailleurs).

Un article qui parle de ça si tu veux : https://javascript.plainenglish.io/how-to-make-your-javascript-code-blazingly-fast-with-this-one-simple-trick-92c53adbf1a1

Reply:
C'est corrigé, merci !